### ISSUE TITLE
Thrown error in plugin does not lift to output

### ISSUE THREAD

**Issue by Strate** [sighjs/sigh#43]
```javascript
export default function(op, opts = {}) {
  if ((!("search" in opts)) || (!("replacement" in opts))) {
    throw new Error("Please provide 'search' and 'replacement' options to 'replace' plugin")
  }

  // rest goes here
```

actual output (with invalid opts):
```
% sigh -j 8 -vv
* running pipelines [ build ] with 8 jobs
```

Expected output should contains error message.

**Comment by ohjames** [comment]
This one is really annoying, gotta fix it.

**Comment by ohjames** [comment]
Released 0.12.28 with the fix.
